### PR TITLE
fix: Update Dashboard readmes to point at policies_v1

### PR DESCRIPTION
**Pull request by jsonpr**
PR for the following issue: https://github.com/cloudquery/cloudquery/issues/4384

### CODE REVIEW

**Review by hermanschaaf — APPROVED** [review]

**Review by yevgenypats — CHANGES_REQUESTED** [review]
I think this should always point to `policies` ? so maybe wait until this PR is merged - https://github.com/cloudquery/cloudquery/pull/4402

**Comment by jsonpr** [comment]
Closing since https://github.com/cloudquery/cloudquery/pull/4402 will do the following:
* Remove deprecated v0 policies from /policies.
* Update /policies with contents of v1 policies currently in /policies_v1.

This will remove the need to point at the policies_v1 folder.

**Pull request closed without merging**
